protos/transaction.proto

```diff
@@ -72,9 +72,9 @@ message Transaction {
     // Fragment IDs are not yet assigned.
     repeated DataFragment fragments = 1;
     // The new schema
-    repeated Field schema = 2;
+    repeated Field schema = 3;
     // Schema metadata.
-    map<string, bytes> schema_metadata = 3;
+    map<string, bytes> schema_metadata = 4;
```

Comment:
Why this change?

Reply:
Oops, not needed anymore.  This was from the old attempt where I was adding `indexes` to an overwrite tx.

Reply:
I reverted this change.  Good catch.